wasi/wasi.go

```diff
@@ -277,16 +323,28 @@ func (w *WASIEnvironment) fd_close(ctx *wasm.HostFunctionCallContext, fd uint32)
 	return ESUCCESS
 }
 
-func args_sizes_get(ctx *wasm.HostFunctionCallContext, argcPtr uint32, argvPtr uint32) (err Errno) {
-	// not implemented yet
-	binary.LittleEndian.PutUint32(ctx.Memory.Buffer[argcPtr:], 0)
-	binary.LittleEndian.PutUint32(ctx.Memory.Buffer[argvPtr:], 0)
-	return 0
+func (w *WASIEnvironment) args_sizes_get(ctx *wasm.HostFunctionCallContext, argsCountPtr uint32, argsBufSizePtr uint32) Errno {
```

Comment:
we are missing wasi_test.go which can be started with unit tests for these two

Reply:
looking at `wasm.HostFunctionCallContext`, it seems simple enough to instantiate so that we can unit test this and below. jit.Test_Simple has some of this kind of thing, and hopefully we'll have more of it soon enough!

Reply:
+1

Reply:
Updated the codes based on your suggestions. PTAL. Thanks!